static/form.js

```diff
@@ -11,34 +11,86 @@ $(document).ready(function () {
 var destinationCounter = 2;
 
 function addDestination() {
-    var additionalDestinationsDiv = document.getElementById('additionalDestinations');
+  
+  var additionalDestinationsDiv = document.getElementById('additionalDestinations');
 
-    // Create a new destination row
-    var destinationRow = document.createElement('div');
-    destinationRow.classList.add('row');
+  // Create a new destination row
+  var destinationRow = document.createElement('div');
+  destinationRow.classList.add('row');
+  // Added an ID for the row, for easy deletion afterwards.
+  destinationRow.id = 'row' + destinationCounter;
 
-    // Create Destination Location input
-    var destinationInput = document.createElement('div');
-    destinationInput.classList.add('col-md-6', 'mb-3');
-    destinationInput.innerHTML = '<label for="destinationLocation' + destinationCounter + '" class="form-label">Destination Location ' + destinationCounter + '</label>' +
+  var destinationInput = document.createElement('div');
+  destinationInput.classList.add('col-md-6', 'mb-3', 'destination');
+  destinationInput.innerHTML = '<label for="destinationLocation' + destinationCounter + '" class="form-label">Destination ' + destinationCounter + '</label>' +
     '<input type="text" class="form-control" id="destinationLocation' + destinationCounter + '" name="destinationLocation' + destinationCounter + '" oninput="updateAirportOptions(\'destinationLocation' + destinationCounter + '\', \'destinationLocationDropdown' + destinationCounter + '\')" onclick="updateAirportOptions(\'destinationLocation' + destinationCounter + '\', \'destinationLocationDropdown' + destinationCounter + '\')">' +
     '<div class="dropdown" id="destinationLocationDropdown' + destinationCounter + '"></div>';
```

Comment:
Running this with the new dropdown menu shows a messed up dropdown menu inside the extra destinations. I believe I know why and will try to fix this before merging.